webservices/utils.py

```diff
@@ -139,6 +138,7 @@ def _get_index_values(self, result):
         """Get index values from last result, to be used in seeking to the next
         page. Optionally include sort values, if any.
         """
+        from webservices.common.models import db
```

Comment:
Was this change pulled in from elsewhere or a part of this changeset explicitly?  I ask because of the logging adjustments that were looked at yesterday, too.

Reply:
Ah I see, thank you!